database/schema/mysql-schema.dump

```diff
@@ -0,0 +1,753 @@
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
```

Comment:
Please remove the schema dump. As we have "non sql only" migrations this is not an option for traewelling.